variables.tf

```diff
@@ -109,3 +82,10 @@ variable "zone_vpc_crn_list" {
   default     = []
   description = "CRN of the VPC for the zones."
 }
+
+variable "enable_cbr" {
+  type        = bool
+  default     = true
+  description = "Flag to enable CBR"
```

Comment:
Don't use acronyms like this. Please use full terminology so its clear to consumers what a CBR is. There is also not enough info here. What CBRs are enabled if set to true?